pa_config/rca.conf

```diff
@@ -93,5 +93,12 @@
     "ClusterTemperatureRca"
   ],
   "muted-deciders": [],
-  "muted-actions": []
+  "muted-actions": [],
+  "bucketization": {
+    "old-gen": {
+      "UNDER_UTILIZED": 10.0,
```

Comment:
Add a comment saying Default Configuration Values are maintained at 2 places. In the code as well as here. We might miss updating at one place if we want to change the defaults in future.

Reply:
The configs are there in the rca.conf although they have the same value as the one we have by default is it gives people an idea of what is configurable just by looking at the rca.conf and not having to read the code. rca.conf takes precedence.

Reply:
What I meant was lets say we want to change the values tomorrow and we just change it in the Conf file and when we fall back to defaults from the code we might fall back to the previous defaults which have been changed. The comment here helps while updating. I dont have a strong opinion regarding this. We can choose to not add comment as well.